### ISSUE TITLE
acs2011_5yr files from URL linked in instructions have different structure than scripts expect

### ISSUE THREAD

**Issue by SkaBob42** [censusreporter/census-postgres#14]
I'm working specifically with these two files:
https://www2.census.gov/acs2011_5yr/summaryfile/2007-2011_ACSSF_By_State_All_Tables/Michigan_All_Geographies_Not_Tracts_Block_Groups.zip
https://www2.census.gov/acs2011_5yr/summaryfile/2007-2011_ACSSF_By_State_All_Tables/Michigan_Tracts_Block_Groups_Only.zip

I'm able to get as far in the instructions as running import_geoheader.sql, but import_sequences.sql seems to expect the data to be in /path/to/acs_files/group1 or /path/to/acs_files/group2. The files from census.gov extract to:

acs2011_5yr/data/tab4/sumfile/prod/2012thru2016/group1/

Am I fetching the wrong files, or should I be alright doing a s/wrong_path/right_path/g on the paths?

**Comment by iandees** [comment]
Instead of re-processing the data, have you tried using the stuff I already built?

http://censusreporter.tumblr.com/post/73727555158/easier-access-to-acs-data

**Comment by SkaBob42** [comment]
Are the data structures in those postgres dumps functionally identical to the ones the census reporter API interacts with? We're working to implement our own copy of the API for one of our sites, because of the throttling on the main census reporter API, and I want to make sure that's what it's going to expect. I already imported the 2016 5yr data from one of those dumps, but I need to import the 2011 5yr data as well.

**Comment by iandees** [comment]
Yes, it's the same data we have in the API.

**Comment by iandees** [comment]
Note that we never offered the older data vintages in our API, so it's not been tested as thoroughly as the newer stuff.

**Comment by SkaBob42** [comment]
Good to know. I'll just use those dumps to import the 2011 data. That's as far back as we're going chronologically. I'll pass that caveat along to our developer. Is that a problem inherent to the census/ACS data itself, or just to the databases used with the API?

**Comment by iandees** [comment]
We didn't offer the older vintage data because there was always newer data to use and we decided to not offer the ability to compare between data vintages because it's inherently difficult to do (geographies and questions change between releases). I went through the process of converting the data in case we wanted to do it in the future, though.

We don't currently offer all the data in the API because we're trying to save money on database storage fees.
